### DISCUSSION TITLE
the device is disconnected from the network

### DISCUSSION THREAD

**Discussion by aliakseika** [openthread/openthread#9092] [Q&A]
Hello. There is the following configuration: the latest version of otbr + CC2652P stick with RCP firmware. I introduce CC1352P device into the network with one of the examples of standard TI provided. After some time, this CC1352P disappears from the network (1-2 days). Maybe someone faced this? is it possible to increase the stability of the connection? Or maybe the problem is just in some settings that I did not take into account?

**Comment by jwhui** [comment]
More information is needed to root cause the issue.

Can you provide log output from both the BR and Thread device? Ideally, the log starts when the device was attached, includes when the device detached from the network, and some time after that.

**Comment by aliakseika** [comment]
no problem, but I still don’t quite understand where to get this log ... all settings and firmware are default. And it will also be quite problematic to choose a log exactly in time, since I can’t predict exactly when the device will disconnect from the network

**Reply by jwhui** [reply]
OTBR is running Linux. You can use the following command to view log output.
```
cat /var/log/syslog | grep otbr-agent
```

**Comment by aliakseika** [comment]
I recorded the entire log, if it is possible to shorten it, then I am ready, but I do not know how to limit it to the required periods. The device was online yesterday, today I noticed that it is not there. The network is still in testing. [log](https://1drv.ms/u/s!AiO65YjMmX8Jiq1HEP2Z_Lysk6AiSw?e=PY65NH) after the reset, it entered the network as if nothing had happened

**Reply by jwhui** [reply]
A quick look at the log, it shows that the child simply stops communicating after some period of time:

```
May 26 02:57:59 ubuntu otbr-agent[1473]: 1d.03:44:53.134 [I] Mle-----------: Child timeout expired
May 26 02:57:59 ubuntu otbr-agent[1473]: 1d.03:44:53.135 [I] Settings------: Read ChildInfo {rloc:0x8001, extaddr:7a49fa192c4b2157, timeout:240, mode:0x0d, version:3}
May 26 02:57:59 ubuntu otbr-agent[1473]: 1d.03:44:53.143 [I] Settings------: Removed ChildInfo {rloc:0x8001, extaddr:7a49fa192c4b2157, timeout:240, mode:0x0d, version:3}
May 26 02:57:59 ubuntu otbr-agent[1473]: 1d.03:44:53.144 [I] Notifier------: StateChanged (0x00000800) [Child-]
May 26 02:57:59 ubuntu otbr-agent[1473]: 1d.03:44:53.144 [I] ChildSupervsn-: Stopping Child Supervision
May 26 17:19:36 ubuntu otbr-agent[1473]: 1d.18:06:30.879 [I] Mle-----------: Receive Child Update Request from child (fe80:0:0:0:7849:fa19:2c4b:2157)
May 26 17:19:36 ubuntu otbr-agent[1473]: 1d.18:06:30.879 [I] Mle-----------: Send Child Update Response to child (fe80:0:0:0:7849:fa19:2c4b:2157)
```

My guess is that the child is getting into an unresponsive state. This is typically a platform-specific issue. Have you tried reaching out to TI for support? Alternatively, you can try a different platform to see if you experience the same.

**Comment by aliakseika** [comment]
Thank you, I'll post a question on the TI forum
